client/src/app/videos/+video-watch/video-watch.component.scss

```diff
@@ -64,6 +64,34 @@ $other-videos-width: 260px;
   }
 }
 
+@keyframes fade {
```

Comment:
Maybe you can put this in the _mixins file

Reply:
>/PeerTube/node_modules/sass-lint/lib/rules/property-sort-order.js:108
              name = prop.first('ident');                      
TypeError: Cannot read property 'first' of null

Reply:
Rofl'

Reply:
> Rofl'

what? I can't commit with those errors, although the css works and looks right. Or is there a way to force commit?

Reply:
You can skip the pre-commit lint-stage with `--no-verify`.

Reply:
so the errors which appear are save to be ignored?

Reply:
It looks more like an error of `sass-lint`, but I don't know much about SASS. If you are sure it is valid, you can bypass the linting.

Reply:
I often have issues with this pre hook and sass-lint, maybe we should remove the hook at least

Reply:
@rigelk I have absolutely no idea about sass/scss, but others say this code is fine and it works, so I assume it must be right.

@Chocobozzz does it have any pros? Generally I am a fan of many tests and lints, as it prevents errors and a bad coding style. But if it only produces errors...